packages/terra-layout/tests/jest/utilities.test.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import { getCustomProps } from '../../src/utilities';
+
+describe('utilities', () => {
+  describe('getCustomProps', () => {
```

Comment:
Should we also test the breakpoints function?

Reply:
There is an issue with the postcss loader (necessary to get the breakpoint values) running with Jest, making it really hard to test the function outright. We could either duplicate the function with hardcoded values and test that, or we could just bank on the responsive nightwatch tests to cover things indirectly. I myself vote for the second option.